hw/top_earlgrey/templates/toplevel.sv.tpl

```diff
@@ -455,7 +479,11 @@ max_intrwidth = (max(len(x.name) for x in block.interrupts)
   % if block.alerts:
 <%
 w = len(block.alerts)
-slice = str(alert_idx+w-1) + ":" + str(alert_idx)
+if 'outgoing_alert' in m:
+  slice = str(outgoing_alert_idx[m['outgoing_alert']]+w-1) + ":" + str(outgoing_alert_idx[m['outgoing_alert']])
+else:
+  slice = str(alert_idx+w-1) + ":" + str(alert_idx)
+endif
```

Comment:
Untested, but maybe this would repeat less?
```python
w = len(block.alerts)
if 'outgoing_alert' in m:
  outgoing_alert = m['outgoing_alert']
  lo = outgoing_alert_idx[outgoing_alert]
else:
  lo = alert_idx
slice = f"{lo+w-1}:{lo}"
```